### ISSUE TITLE
New requirement wrt netinfo 

### ISSUE THREAD

**Issue by playfulThinking** [martinmoec/fable-react-native-how-to#2]
Because the latest version of React Native has moved netinfo out of the base installation, and this is referenced in https://github.com/fable-compiler/fable-react-native, it appears you also have to do this (for iOS):

```
npm install --save @react-native-community/netinfo
npx pod install
```

There is more you have to do for Android, which I haven't done yet. Full instructions are here: https://github.com/react-native-community/react-native-netinfo

**Comment by martinmoec** [comment]
Hi @fremontmj,Thanks for the heads up! I have included the module in 8c6e340 and it works for both iOS and Android.

**Comment by playfulThinking** [comment]
Thank you! Your how-to helped me a lot.
